.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+matrix:
+  include:
+    - python: 2.7
+      env: PLONE_VERSION=4
+    - python: 2.7
+      env: PLONE_VERSION=5
+    - python: 3.7
+      env: PLONE_VERSION=5
+install:
+  - mkdir -p buildout-cache/downloads
+  - cp requirements-$PLONE_VERSION.x.txt requirements.txt
+  - python bootstrap.py
```

Comment:
What about dropping `bootstrap.py` using `pip install -r requirements-$PLONE_VERSION.x.txt`?

Reply:
I tried that (in another repo I think) and afterwards it didn't find bin/buildout. Didn't take the time to debug.